### ISSUE TITLE
Tables and spaces - and some problems

### ISSUE THREAD

**Issue by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1813]
Unfortunately, the use of multiple spaces in tables to align them creates certain problems.

For example, when trying to find and remove double spaces, when searching for single words, etc.

Can you replace spaces in tables with a different alignment method?

Thank you :)

**Comment by introt** [comment]
Do you have any suggestions? I'm not seeing huge gains from any alternatives, while all of them would break backwards compatibility and most wouldn't be hand-editable.

> when trying to find and remove double spaces

The parser doesn't care too much about alignment, it's the `|`s where the magic is at. Also, Zim's own find-and-replace should ignore formatting, so you can use that to be extra safe.

> when searching for single words

Can you elaborate on how the multiple spaces create problems here? Doesn't sound like anything a little regex couldn't handle ;)

**Comment by Nick222** [comment]
I don't think you need to force the average user Zim to learn regular expressions.
Do not forget that the simplicity of the format of this program makes it very easy (technically and psychologically) to edit its files using third-party software - which potentially leads to conflicts between different edits.

In addition, it is very difficult to work with line breaks in table cells, which, in my opinion, requires its own solution.
In general, as a user, the implementation of tables in Zim gives me a feeling of ill-conceived and unfinished business.

I think that the source text of the note needs a label for the beginning of the table and separators of table cells, and formatting (including alignment) should be done by the program itself.

If you want, we can start a discussion about the use of tables in Zim, especially since there are already a lot of complaints about this option ...
